### PR TITLE
Add name to student model

### PR DESCRIPTION
**Pull request by alanhala**
Hey @jmtast 

In this PR I created a failing test for the Student's name. This test was fixed by adding a column to the table.

Please check it out and let me know if everything is OK. Thanks!

### CODE REVIEW

File: spec/models/student_spec.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require 'spec_helper'
+
+describe Student do 
+  it 'has a name' do
+    expect(Student.new(name: 'Alan Halatian').name).to eq('Alan Halatian')
+  end
+end
```
Comment (jmtast) [inline spec/models/student_spec.rb]: Extra end line is missing.

File: db/migrate/20150803231647_add_name_to_students.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class AddNameToStudents < ActiveRecord::Migration
+  def change
+    add_column :students, :name, :string
+  end
+end
```
Comment (jmtast) [inline db/migrate/20150803231647_add_name_to_students.rb]: Here too, no extra end line.

File: db/migrate/20150720021243_add_name_to_subjects.rb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-class AddNameToSubject < ActiveRecord::Migration
+class AddNameToSubjects < ActiveRecord::Migration
   def change
     add_column :subjects, :name, :string
   end
```
Comment (jmtast) [inline db/migrate/20150720021243_add_name_to_subjects.rb]: Please add one in this file too, that's my bad.

**Comment by alanhala** [comment]
@jmtast done!

**Comment by jmtast** [comment]
:shipit:

**Pull request merged**